StRoot/StRHICfUtil/StRHICfFunction.cxx

```diff
@@ -22,8 +22,11 @@ void StRHICfFunction::initChecker()
   mGSOBarNum = 0;
   mGSOBarTableNum = 0;
   mPlateNum = 0;
-  memset(mValueForRecoNum, 0, sizeof(mValueForRecoNum));
-  memset(mLeakageNum, 0, sizeof(mLeakageNum));
+  
+  for(int i=0; i<5; i++){
+    if(i<3){mLeakageNum[i] = 0;}
+    mValueForRecoNum[i] = 0;
+  }
```

Comment:
Why this change?
In C++ you probably want to zero C-arrays like this:

```suggestion
std::fill(mLeakageNum, mLeakageNum+3, 0);
std::fill(mValueForRecoNum, mValueForRecoNum+5, 0);
```

Reply:
Thank you for the suggestion.
I thought FOR loop was more effective.
I commit again